apps/jobbrowser/src/jobbrowser/conf.py

```diff
@@ -77,29 +77,32 @@
   default=False
 )
 
-ENABLE_QUERIES_LIST = Config(
-  key="enable_queries_list",
-  help=_("Show the Queries section for listing Hive/Impala query history and providing more troubleshooting information."),
-  type=coerce_bool,
-  default=False
-)
-
 ENABLE_HISTORY_V2 = Config(
   key="enable_history_v2",
   help=_("Show the version 2 of job/query History which unifies the all into one."),
   type=coerce_bool,
   default=False
 )
 
+def query_store_url_is_set():
+  """Check if query store url is configured"""
+  return QUERY_STORE.SERVER_URL.get() != ''
+
 QUERY_STORE = ConfigSection(
   key="query_store",
-  help=_("""Credentials for query store API."""),
+  help=_("Configs for managing query store interface."),
   members=dict(
     SERVER_URL=Config(
       key="server_url",
-      default='http://localhost:8080/',
+      default='',
       help=_("URL of Query Store API server.")
-    )
+    ),
+    ENABLE=Config(
```

Comment:
Suggestion: This config is called `IS_ENABLE` for other services in Hue. For consistency, should be apply it here too and change its references?

Reply:
Sure, can change it to is_enabled.

Reply:
Done.